de.dlr.sc.virsat.model.edit/src/de/dlr/sc/virsat/model/dvlm/categories/propertyinstances/provider/DVLMValuePropertyInstanceItemProvider.java

```diff
@@ -39,6 +41,13 @@ public String getText(Object object) {
 		if (object instanceof ValuePropertyInstance) {
 			ValuePropertyInstance vpi = (ValuePropertyInstance) object;
 			String value = vpi.getValue() == null ? "" : vpi.getValue();
+
+			//If string is multiline then just show the first line
+			String lineSeperator = System.getProperty(LINE_SEPERATOR_PROPERTY);
+			if (value.indexOf(lineSeperator) > 0) {
+				value = value.substring(0, value.indexOf(lineSeperator));

```

Comment:
Should we maybe add three dots if there are more lines?

Reply:
Good idea, i agree to that.